lib/omni_auth/strategies/email.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Based on https://github.com/weg-li/weg-li/blob/master/app/lib/omni_auth/strategies/email.rb
+# Original author: https://github.com/phoet
+
+require 'base64'
+require 'English'
+
+module OmniAuth
+  module Strategies
+    class Email
+      include OmniAuth::Strategy
+
+      def request_phase
+        redirect '/sessions/email'
+      end
+
+      def callback_phase
+        token = request.params['token']
+        fail!(:authenticity_error) if token.blank?
+
+        begin
+          decoded_token = EmailAuthToken.decode(token)
+        rescue StandardError
+          fail!(:authenticity_error, $ERROR_INFO)
```

Comment:
```suggestion
        rescue StandardError => error
          fail!(:authenticity_error, error)
```
and we can skip the `require 'English'`